blocksync/buffer.go

```diff
@@ -65,7 +66,7 @@ func (b *blockBuffer) Flush(blk *blockchain.Block) (bool, bCheckinResult) {
 			break
 		}
 		delete(b.blocks, heightToSync)
-		if err := commitBlock(b.bc, b.ap, blk); err != nil {
+		if err := commitBlock(b.bc, b.ap, blk); err != nil && err != db.ErrAlreadyExist {
```

Comment:
if err == db.ErrAlreadyExist, we should also not say "Successfully committed block". Maybe ether skip the log or say "Already exists"

Reply:
emm, looks better